python/rikai/mixin.py

```diff
@@ -47,6 +47,14 @@ def to_pil(self) -> "PIL.Image.Image":
         pass
 
 
+class ToDict(metaclass=ABCMeta):
```

Comment:
Why use ABCMeta instead of just ABC here like the rest of the codebase?

Reply:
muscle memory